mipengine/node/tasks/views.py

```diff
@@ -46,18 +48,50 @@ def create_view(
         A list of dataset names
     columns : List[str]
         A list of column names
-    filters_json : str(dict)
+    filters : dict
         A Jquery filters object
 
     Returns
     ------
     str
-        The name of the created view in lower case
+        The name of the created view
     """
     view_name = create_table_name(
         "view", command_id, context_id, node_config.identifier
     )
-    views.create_view(
+    views.create_pathology_view(
         view_name=view_name, pathology=pathology, datasets=datasets, columns=columns
     )
     return view_name.lower()
+
+
+@shared_task
+def create_view(
+    context_id: str, command_id: str, table_name: str, columns: List[str], filters: dict
+) -> str:
+    """
+    Creates a view of a table with specific columns and filters to the DB.
+
+    Parameters
+    ----------
+    context_id : str
+        The id of the experiment
+    command_id : str
+        The id of the command that the view
+    table_name : str
+        The name of the table
+    columns : List[str]
+        A list of column names
+    filters : dict
+        A Jquery filters object
+
+    Returns
+    ------
+    str
+        The name of the created view
+    """
+    view_name = create_table_name(
+        "view", command_id, context_id, node_config.identifier
+    )
+    views.create_view(view_name=view_name, table_name=table_name, columns=columns)
```

Comment:
filters ignored?

Reply:
This branch is different from the dev/MIP-18/Add-the-possibility-of-filters-when-creating-a-view.
Filters will be added on the MIP-18.